python/cuopt/cuopt/linear_programming/problem.py

```diff
@@ -0,0 +1,796 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.  # noqa
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+import numpy as np
+
+import cuopt.linear_programming.data_model as data_model
+import cuopt.linear_programming.solver as solver
+import cuopt.linear_programming.solver_settings as solver_settings
+
+
+# The type of a variable is either continuous or integer
+class VType(str, Enum):
+    CONTINUOUS = "C"
```

Comment:
Lets add definition to these similar to https://github.com/NVIDIA/cuopt/blob/455bdb0b4f99b64e978f01b44fe37d7dd558308a/python/cuopt_self_hosted/cuopt_sh_client/thin_client_solver_settings.py#L46

Reply:
Thanks for updating